### ISSUE TITLE
"Write a Message" pane gets stuck open

### ISSUE THREAD

**Issue by jfrederickson** [henryboldi/felony#65]
I just had this happen after clicking on my two buddy entries a few times:

![screen shot 2016-07-12 at 1 07 14 pm](https://cloud.githubusercontent.com/assets/1923178/16776116/d3f2e5ce-4831-11e6-9611-dd4858063f16.png)

**Comment by jfrederickson** [comment]
Looks like this happens when rapidly clicking on a buddy in the list.

**Comment by greg-js** [comment]
Confirmed.

I did a little testing, and it seems to happen when the animation for bringing up the opener in lines 48-55 of `./app/components/keychain/KeyChainComposerOpener.js` has not yet completed by the time the user deselects the last selected buddy.

If the animation duration is set to 1, the problem disappears, but obviously then the animation disappears too. I couldn't immediately find the code that removes the opener when no buddies are selected, but I'm guessing it should be a relatively easy fix once one does. I might look into it later tonight if no one else has by then.
